tests/unit_tests/navier_stokes_unit_test/CMakeLists.txt

```diff
@@ -178,3 +178,39 @@ foreach(dim RANGE 1 3)
 
 endforeach()
 
+set(TEST_SRC
+    reynolds_averaged_navier_stokes_sa_neg_manufactured_solution_source.cpp
```

Comment:
Hmm, has the file `reynolds_averaged_navier_stokes_sa_neg_manufactured_solution_source.cpp` been committed and pushed yet? I am unable to see it (might be due to the update in gitignore).

Reply:
fixed